packages/now-node/test/fixtures/10-node-engines/vercel.json

```diff
@@ -1,15 +0,0 @@
-{
-  "version": 2,
-  "builds": [
-    {
-      "src": "**/*.js",
-      "use": "@vercel/node"
-    }
-  ],
-  "probes": [
-    { "path": "/empty", "mustContain": "RANDOMNESS_PLACEHOLDER:14" },
-    { "path": "/greater", "mustContain": "RANDOMNESS_PLACEHOLDER:14" },
-    { "path": "/major", "mustContain": "RANDOMNESS_PLACEHOLDER:12" },
-    { "path": "/range", "mustContain": "RANDOMNESS_PLACEHOLDER:12" }
-  ]
-}
```

Comment:
Same issue here, this file should not have been deleted. Take a look at PR #5925

Reply:
File is back now